modules/irmaseal-pkg-server.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.irmaseal-pkg;
+in
+{
+
+  options.services.irmaseal-pkg = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        This option enables the IRMAseal-PKG server.
+      '';
+    };
+
+    package = mkOption {
+      type = types.path;
+      default = pkgs.irmaseal-pkg;
+      description = "The IRMAseal-PKG package to use";
+    };
+
+    host = mkOption {
+      type = types.str;
+      default = "0.0.0.0";
```

Comment:
Perhaps this should be "127.0.0.1"
My understanding is that 0.0.0.0 is accessible from outside, and so 127 is a more secure default.

Reply:
Hmm, you are right, I think I'd prefer 127.0.0.1, I just defaulted to what `irmaseal-pkg` defaults to:

```
irmaseal-pkg-server
run the IRMAseal PKG HTTP server

USAGE:
    irmaseal-pkg server [OPTIONS]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

OPTIONS:
    -H, --host <HOST>        host to bind this service to [default: 0.0.0.0]
    -i, --irma <irma>        url of the IRMA go server to use for authentication [default: https://irma-
                             noauth.demo.sarif.nl]
    -p, --port <PORT>        TCP port to bind this service to [default: 8087]
    -P, --public <public>    path to the public key [default: ./pkg.pub]
    -S, --secret <secret>    path to the private key [default: ./pkg.sec]

```